cli/npm/cache.rs

```diff
@@ -188,14 +187,14 @@ impl ReadonlyNpmCache {
   ) -> PathBuf {
     if id.copy_index == 0 {
       self.package_folder_for_name_and_version(
-        &id.name,
-        &id.version,
+        &id.nv.name,
+        &id.nv.version,
```

Comment:
Some changes that could be done (like only providing `&id.nv` here), will be in the next deno_graph refactor PR.